Extension/src/common.ts

```diff
@@ -897,6 +913,7 @@ export function downloadFileToStr(urlStr: string, headers?: OutgoingHttpHeaders)
     });
 }
 
+/** CompilerPathAndArgs retains original casing of text input for compiler path and args */
```

Comment:
Is this supposed to be `/*`?

Reply:
The comment doesn't show for `/*`, it needs to be `/**`.

Reply:
Do you know why? I didn't see `/**` being used in other cases.

Reply:
Ah, I see now -- they're all like /** -- I was seeing a /* before for some reason.